include/internal/symhacks.h

```diff
@@ -23,4 +23,4 @@
 
 # endif
 
-#endif                          /* ! defined HEADER_VMS_IDHACKS_H */
+#endif                          /* ! defined OPENSSL_VMS_IDHACKS_H */
```

Comment:
Sorry, but I missed this before, apparently this comment should be ! defined OPENSSL_INTERNAL_SYMHACKS_H ?